api/PopStateEvent.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "PopStateEvent": {
+        "__compat": {
+          "description": "<code>PopStateEvent()</code> constructor",
+          "spec_url": "https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-popstateevent-interface",
```

Comment:
@sideshowbarker for constructors with nothing in particular to link to, do you think we should do this, or just omit the spec URL?

Reply:
> @sideshowbarker for constructors with nothing in particular to link to, do you think we should do this

I have a slight preference for doing that — though I agree it’s not a very friendly place to drop developers into.

> or just omit the spec URL?

If we omit the spec URL completely, then we’re not giving developers any help at all if they want to confirm what the spec says are the arguments for the constructor.

The IDL at least minimally shows what the arguments for the constructor are, so it seems like that makes the spec link marginally useful.

Reply:
OK, so let's link to the most useful thing we can find then, if the spec defines the constructor at all.